httpbin/cmd/cmd.go

```diff
@@ -74,10 +74,20 @@ func mainImpl(args []string, getEnv func(string) string, getHostname func() (str
 		logger = slog.New(handler)
 	}
 
+	env := make(map[string]string)
+	for _, e := range os.Environ() {
+		v := strings.SplitN(e, "=", 2)
```

Comment:
Minor, but I'd write this as

```go
key, val, _ := strings.Cut(e, "=")
```

just because I prefer giving things names rather than referring to opaque slice indexes where possible/convenient.

Reply:
Good idea, such 'unpaking tuples' is also my preference, I've just failed to find Go-idiomatic way - I'm a Go newbie.
Done.